addons/mail/static/src/messaging/component/chatter_topbar/chatter_topbar.scss

```diff
@@ -54,11 +62,16 @@
 .o_ChatterTopbar .o-active {
     color: $o-brand-odoo;
     background-color: lighten(gray('300'), 7%);
-    border-color: gray('300');
+    border-right-color: gray('300');
+    border-left-color: gray('300');
     border-bottom-color: transparent;
     border-top-color: transparent;
 }
 
+.o_ChatterTopbar_button:hover {
```

Comment:
There is something strange with the text color, it changes on hover when the element is not active, but it doesn't change on hover when the element is active.

Reply:
And what do you propose as color for text hover if button is active ?

Reply:
I would say the same color on hover whether the tab is active or not (basically hover taking priority over normal style the same way in both cases).